modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json

```diff
@@ -0,0 +1,197 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/bbp/core/entity/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpentsh": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/",
+      "bbpqsh" : "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "bbplocsh": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/",
+      "this" : "{{base}}/schemas/bbp/experiment/patchedcell/v0.1.0/shapes/"
+    }
+  ],
+
+  "@type" : "owl:Ontology",
+  "imports" : [ "{{base}}/schemas/bbp/core/entity/v0.1.0",
+                "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0",
+                "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0",
+                "{{base}}/schemas/bbp/core/brainlocation/v0.1.0"],
+  "shapes" : [
+    {
+      "@id": "this:PatchedCellShape",
+      "@type": "sh:NodeShape",
+      "label": "Patched cell shape definition",
+      "targetClass": "nsg:PatchedCell",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "bbpentsh:EntityShape"
+        },
+        {
+          "node": "bbplocsh:BrainLocationShape"
```

Comment:
What is the best option for BrainLocationShape usage? Two options for reusing the  brain location properties:
* rooted in the PatchedCell entity 
* rooted in a brainLocation property which is itself rooted in the PatchedCell entity

We need to be consistent on the way the BrainLocationShape is enforce across the different BBP domains.

Thoughts ?

Reply:
I will change it so it is rooted in a brainLocation property which is itself rooted in the PatchedCell entity

Reply:
Excellent.